specs/ScreenCaptureRequested

```diff
@@ -0,0 +1,394 @@
+  /// The associated frame information that requests the screen capture  
+  /// permission. This can be used to grab the frame source, name, frameId,  
+  /// and parent frame information. 
+  [propget] HRESULT FrameInfo([out, retval] ICoreWebView2FrameInfo**  
+     frameInfo); 
+  /// By default, both the `ScreenCaptureRequested` event handlers on the 
+  /// `CoreWebView2Frame` and the `CoreWebView2` will be invoked, with the 
+  /// `CoreWebView2Frame` event handlers invoked first. The host may 
+  /// set this flag to `TRUE` within the `CoreWebView2Frame` event handlers 
+  /// to prevent the remaining `CoreWebView2` event handlers from being  
+  /// invoked. 
+  /// 
+  /// If a deferral is taken on the event args, then you must synchronously 
+  /// set `Handled` to TRUE prior to taking your deferral to prevent the 
+  /// `CoreWebView2`s event handlers from being invoked. 
+  [propget] HRESULT Handled([out, retval] BOOL* handled); 
+  /// Sets the `Handled` property. 
+  [propput] HRESULT Handled([in] BOOL handled); 
+  /// The host may set this flag to cancel the screen capture. If canceled, 
+  /// the screen capture UI is not displayed regardless of the 
+  /// `Handled` property. 
+  [propget] HRESULT Cancel([out, retval] BOOL* cancel); 
+  /// Sets the `Cancel` property. 
+  [propput] HRESULT Cancel([in] BOOL cancel); 
+  /// Returns an `ICoreWebView2Deferral` object. Use this deferral to 
+  /// defer the decision to show the Screen Capture UI. 
+  /// 
+  /// Returns an `ICoreWebView2Deferral` object.    
+  HRESULT GetDeferral([out, retval] ICoreWebView2Deferral** deferral); 
+} 
+/// This is an extension of the ICoreWebView2Frame interface that supports ScreenCaptureRequested 
+// MSOWNERS: stevenwei@microsoft.com 
+[uuid(12885cda-9caa-4793-9c38-f15827dbab1f), object, pointer_default(unique)] 
+interface ICoreWebView2Frame3 : IUnknown { 
+  /// Add an event handler for the `ScreenCaptureRequested` event. 
+  /// `ScreenCaptureRequested is raised when content in an iframe any of its 
+  /// descendant iframes requests permission to use the Screen Capture  
+  /// API from getDisplayMedia() 
+  /// 
+  /// This relates to the `ScreenCaptureRequested` event on the  
+  /// CoreWebView2`. 
+  /// Both these events will be raised in the case of an iframe requesting 
+  /// permission. The `CoreWebView2Frame`'s event handlers will be invoked 
+  /// before the event handlers on the `CoreWebView2`. If the `Handled`  
+  /// property of the `ScreenCaptureRequestedEventArgs` is set to TRUE  
+  /// within the`CoreWebView2Frame` event handler, then the event will not  
+  /// be raised on the `CoreWebView2`, and its event handlers will not be   
+  /// invoked. 
+  /// 
+  /// In the case of nested iframes, the 'ScreenCaptureRequested' event will 
+  /// be raised from the top level iframe. 
+  /// 
+  /// If a deferral is not taken on the event args, the subsequent scripts  
```

Comment:
Is this right? Subsequent scripts aren't blocked are they? I would have thought all that is blocked is async completion of getDisplayMedia method?